e2e/app/sidebar/runtime-panel.ts

```diff
@@ -282,14 +325,20 @@ export default class RuntimePanel extends BaseSidebar {
     logger.info('Unattached persistent disk is deleted');
   }
 
-  async applyChanges(): Promise<NotebookPreviewPage> {
+  async applyChanges(expectPreviewPageNavigate = false): Promise<NotebookPreviewPage | null> {
     await this.clickButton(LinkText.Next);
     await this.clickButton(LinkText.ApplyRecreate);
     await this.waitUntilClose();
 
-    // Automatically opens the Preview page
-    const notebookPreviewPage = new NotebookPreviewPage(this.page);
-    await notebookPreviewPage.waitForLoad();
+    // The preview page automatically opens only if the browser is currently on a
+    // notebook / terminal page.
+    let notebookPreviewPage = null;
```

Comment:
`let notebookPreviewPage:  NotebookPreviewPage = null;` makes eslint happier.
`ESLint: Unsafe member access .waitForLoad on an any value. (@typescript-eslint/no-unsafe-member-access)`

Reply:
Done, but FYI I am pretty much ignoring the linter as it generates hundreds of warnings currently. My suggestions on how to make the linter useful are here: https://precisionmedicineinitiative.atlassian.net/browse/RW-7698

Reply:
I've been ignoring eslint warnings as well because too many. I need find time to work on RW-7698. :)